pkgs/by-name/ld/ldap-auth-sh/package.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, curlFull
+, openldap
+, coreutils
+, gnused
+, gnugrep
+}:
+
+stdenv.mkDerivation {
+  pname = "ldap-auth-sh";
+  version = "unstable-2019-02-23";
```

```suggestion
  version = "0-unstable-2019-02-23";
```

versions must start with a digit